### DISCUSSION TITLE
Can't content Content rectransform of Grid scroll at Upper Center

### DISCUSSION THREAD

**Discussion by hieutran196** [Brian-Jiang/UnlimitedScrollUI#14] [Q&A] [answered]
First of all, thank you for your great asset!
But I can't create Item at Uper center, Content auto move top left inside Viewport when generate.
Can I make it center?
<img width="1630" alt="DemoGrid - AGA-Client - Android - Unity 2020 3 25f1 Personal (Personal)  Metal  2022-01-08 21-46-23" src="https://user-images.githubusercontent.com/45929283/148648556-eb683051-fc1a-4d58-b259-4ac010fe6707.png">

**Comment by Brian-Jiang** [comment, marked as answer]
Hi,
Unfortunately, I used the anchored position of content to calculate first row and column that's being displayed, so it needs to be at upper left. However, you can create another "holder" object as child of ScrollView and parent of Viewport, then you can set that to center (but you can't set "auto hide and expand viewport" to both scrollbar visibility property in scroll view. I'll consider adding a setting so you can easily create items at center in next version. Thanks for your support, hope this can help.
<img width="1920" alt="2022-01-09" src="https://user-images.githubusercontent.com/35553805/148685962-25e4811c-2500-4ae7-b56d-1b9b4b435105.png">

**Reply by hieutran196** [reply]
Cool, thanks.
It will be greater if we have more features:
 - Jump to item
 - We can remove/add one item in grid and reload data
